packages/docs/pages/ButtonGroup/ButtonGroupPage.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box, ButtonGroup, H0, H1, Text } from '@bigcommerce/big-design';
+import React from 'react';
+
+import { Code, CodePreview } from '../../components';
+import { ButtonGroupPropTable } from '../../PropTables/ButtonGroupPropTable';
+
+const ButtonGroupPage = () => (
+  <>
+    <H0>Button Group</H0>
+
+    <Text>
+      The <Code primary>Button Group</Code> component is used for grouping actions like <Code primary>Button</Code>
```

Comment:
This looks weird to me.

```suggestion
      The <Code primary>Button Group</Code> component is used for grouping actions as <Code primary>Button</Code>
```

Maybe?